docs/swarm/swarm-tutorial/rolling-update.md

```diff
@@ -67,6 +78,16 @@ applies the update to nodes according to the `UpdateConfig` policy:
     redis
     ```
 
+    The scheduler applies rolling updates as follows:
+
+    * Stop the initial number of tasks according to `--update-parallelism`.
+    * Schedule updates for the stopped tasks.
+    * Start the containers for the updated tasks.
+    * Wait for the health checks on individual tasks to return a state of either
```

Comment:
@cpuguy83 should we temporarily remove the reference to health checks? I think there's still work to be done on those, so perhaps removing it for now is better than to give false expectations